host.go

```diff
@@ -378,6 +387,60 @@ func (h *Host) InitCommands(c *chat.Commands) {
 		},
 	})
 
+	c.Add(chat.Command{
+		Prefix:     "/nick",
+		PrefixHelp: "NAME",
+		Help:       "Rename yourself.",
+		Handler: func(room *chat.Room, msg message.CommandMsg) error {
+			args := msg.Args()
+			if len(args) != 1 {
+				return errors.New("missing argument")
+			}
+			u := msg.From()
+
+			member, ok := room.MemberByID(u.ID())
+			if !ok {
+				return errors.New("failed to find member")
+			}
+
+			oldID := member.ID()
+			newID := sanitize.Name(args[0])
+
+			if newID == oldID {
+				return errors.New("new name is the same as the original")
+			}
+
+			// check nick registration
+			if h.auth.keynamesMode {
+				identity, identified := member.Identifier.(*Identity)
+				if identified {
+					identity.SetSymbol(strings.Replace(identity.symbol, "✓", "", -1 ))
```

Comment:
We shouldn't hardcode symbols here. If we want a checkmark symbol for registered names, we can have them in the key comment, right?

We can have cases where people wish to reserve names but not want to have a checkmark, or maybe a checkmark is earned in some other way on some servers.

Reply:
Given how contentious this issue has been, the symbol prefix may also be the wrong place to do this. 

Instead, what if we added a field to the /whois listing that displays the ssh key comment, in it's entirety?

The situation I'm trying to avoid by enforcing the symbol appearance is one where a user claims their nick is reserved, when it isn't. Other non-admin users should be able to depend on it, so long as they can trust the operators (who can still assign any symbol string with `/rename`). 

If we put this in a `/whois` field, instead, other users will always have access to that, and will always be able to depend on it, whether the user has a registered nick or not. I can then undo all the symbol-setting business; operators can continue to `/rename` with any symbols they want,  and future developers can brainstorm and implement new features there as they please.